content/concepts/webpack-config.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Webpack Config
```

Comment:
Perhaps rename this as `concepts/configuration.md`. No need to mention webpack at the title as it's given through context.